### ISSUE TITLE
Docs improvement suggestions - was: Issues to get extensions running with a standard basic react-redux setup

### ISSUE THREAD

**Issue by peletiah** [zalmoxisus/redux-devtools-extension#161]
TLDR: Meanwhile the extension is working with my app, but I'll still file this report if you consider to extend the docs at some point so they work better for newbs like me :-) You can close the issue if it's irrelevant.

Also where does initialState come from? 

And the links to commits in Section 2 where confusing and didn't make things any clearer (Only after the fact).

Hope I don't sound ungrateful, love the extension, thank you for it! And after all, I figured out how to get it to work based on the docs and examples. I'd contribute to the docs, but I'm afraid I lack the insight to improve it on a professional level.

=====

It's late, I'm new to redux and react, I've been fighting all day with the weirdly intricate [redux-form-docs](http://redux-form.com/5.3.1/#/getting-started) and I need redux-devtools to figure out what I'm doing wrong with my forms. But now I can't seem to get the configuration for your extension right, duh!

To illustrate my difficulties with the [implementation](https://github.com/zalmoxisus/redux-devtools-extension#implementation)-description, I'll retrace my steps and my mistakes:

I have started from a working store, which initially looked like this:

```javascript
const reducers = combineReducers({
  routeState: routeReducer,
  menuState: menuReducer,
  form: formReducer
});

const store = createStore(reducers);

export default store;
```

I replaced that with configureStore() according to step 2 (Without enhancers and middleware). First I didn't add ```initialState```, as I didn't realize where it's coming from and thought it's another part like middleware I don't use.
```javascript
export default function configureStore(initialState) {
  const store = createStore(
    reducers,
    initialState,
    window.devToolsExtension && window.devToolsExtension()
  );
  return store;
}
```
 I still don't know where initialState is coming from and what it is, but after adding it, I was a step further.

After some more errors and with the help of your examples, I figured out that I have to call configureStore() to get a proper store-object for my Provider-component:

```javascript
function configureStore(initialState) {
  const store = createStore(
    reducers,
    initialState,
    window.devToolsExtension && window.devToolsExtension()
  );
  return store;
}

const store = configureStore();

export default store
```

```javascript
[..]
import store from './store/Store'

render(
  <Provider store={store}>
    <Router history={browserHistory} routes={routes}/>
  </Provider>,
    document.querySelector('#main')
)
```

Now I no longer get any error-messages, but the extension is still blank, except for mocking me to "Make sure to follow the [instructions.](No store found. Make sure to follow the instructions.)"

After a few tab-switches to this tab and back to my app and no further changes in the code, the extension suddenly works. Cool!

**Comment by zalmoxisus** [comment]
Thanks for the suggestion. Rewriting the docs would be great. And also to add links to Redux docs where it is necessary, for example in your case about [createStore](http://redux.js.org/docs/api/createStore.html).

And BTW, in you initial code you didn't need any changes and `initialState`, according to Redux docs, is optional. It would be just:

```js
const reducers = combineReducers({
  routeState: routeReducer,
  menuState: menuReducer,
  form: formReducer
});

const store = createStore(reducers, window.devToolsExtension && window.devToolsExtension());

export default store;
```

**Comment by peletiah** [comment]
>const store = createStore(reducers, window.devToolsExtension && window.devToolsExtension());

>export default store;

Oh! Well that's a lot easier then!

Thanks for pointing out the links to the redux-docs in regards to [initialState](http://redux.js.org/docs/api/createStore.html). Now I realize ```initialState``` is what I feed my reducers in order to define the objects I feed to them and the store. Yep, those links would make sense :-)
